packages/bluesnap-direct-integration/src/bluesnap-direct-constants.ts

```diff
@@ -0,0 +1,62 @@
+import { HostedFieldType } from '@bigcommerce/checkout-sdk/payment-integration-api';
+
+import { BlueSnapDirectHostedFieldTagId as HostedFieldTagId } from './types';
+
+export const BlueSnapHostedFieldType: {
+    [HostedFieldTagId.CardNumber]: HostedFieldType.CardNumber;
+    [HostedFieldTagId.CardExpiry]: HostedFieldType.CardExpiry;
+    [HostedFieldTagId.CardName]: HostedFieldType.CardName;
+    [HostedFieldTagId.CardCode]: HostedFieldType.CardCode;
+} = {
+    [HostedFieldTagId.CardNumber]: HostedFieldType.CardNumber,
+    [HostedFieldTagId.CardExpiry]: HostedFieldType.CardExpiry,
+    [HostedFieldTagId.CardName]: HostedFieldType.CardName,
+    [HostedFieldTagId.CardCode]: HostedFieldType.CardCode,
+};
```

Comment:
```suggestion
export const BlueSnapHostedFieldType = {
    [HostedFieldTagId.CardNumber]: HostedFieldType.CardNumber,
    [HostedFieldTagId.CardExpiry]: HostedFieldType.CardExpiry,
    [HostedFieldTagId.CardName]: HostedFieldType.CardName,
    [HostedFieldTagId.CardCode]: HostedFieldType.CardCode,
};
```

Reply:
Is there any reason you put type information for a const?

Reply:
I use `BlueSnapHostedFieldType` to map types between BlueSnap and BigCommerce, so I need to be this specific. If I commit your suggestion TS would infer that any key of `BlueSnapHostedFieldType` returns a `HostedFieldType`:

```
const BlueSnapHostedFieldType: {
    ccn: HostedFieldType;
    exp: HostedFieldType;
    noc: HostedFieldType;
    cvv: HostedFieldType;
}
```

This gives me errors because `HostedFieldType.CardCodeVerification` and `HostedFieldType.CardNumberVerification` does not exist on my mappings. But, if I explicitly exclude them everything is fine:

```
const BlueSnapHostedFieldType: {
    ccn: HostedFieldType.CardNumber;
    exp: HostedFieldType.CardExpiry;
    noc: HostedFieldType.CardName;
    cvv: HostedFieldType.CardCode;
}
```

Sorry, I haven't found a more elegant way to do this mapping, please let me know if you have a better idea.